functests/performance/rt-kernel.go

```diff
@@ -59,4 +60,20 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 	})
 
+	It("[test_id:28526][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Non worker-cnf node should not have RT kernel installed", func() {
+
+		By("Skipping test if cluster does not have another available worker node")
+		nonRTWorkerNodes, err := nodes.GetNonRTWorkers()
+		Expect(err).ToNot(HaveOccurred())
+
+		if len(nonRTWorkerNodes) == 0 {
+			Skip("Skipping test because there are no additional non-cnf worker nodes")
+		}
+
+		cmd := []string{"uname", "-a"}
+		execOut := Expect(execCommandOnWorker(cmd, &nonRTWorkerNodes[0]))
```

Comment:
I expected more like this, I think it's better readable:
```
kernel := execCommandOnWorker(cmd, &nonRTWorkerNodes[0])
Expect(kernel).To...
Expect(kernel).NotTo...
```
But yours will work as well, I leave it up to you if want to change it.